README.md

```diff
@@ -31,44 +31,47 @@ The output is uploaded to Azure Blob Storage, a suitable connection string shoul
 For further details see [Azure Blob Storage](https://azure.microsoft.com/en-gb/services/storage/blobs/).
 
 The ETL retrieves all the Sexual Health and Information Services from the Syndication API via the `modifiedsince` end point.
```

Comment:
This section will need updating once the changes to use the `all` endpoint has been made.
Can the references to Serco be removed please.